eiffel/src/main/java/com/etiennelenhart/eiffel/viewmodel/EiffelViewModel.kt

```diff
@@ -111,3 +154,9 @@ abstract class EiffelViewModel<S : State, A : Action>(
         scope.cancel()
     }
 }
+
+internal fun <S : State, A : Action> EiffelViewModel<S, A>.log(message: String) {
```

Comment:
I think this should either be moved next to the `Eiffel` object or simply put into `EiffelViewModel` as a `private fun` without receiver.